minard/shifter_information.py

```diff
@@ -0,0 +1,97 @@
+import psycopg2
+from .db import engine
+from .views import app
+from wtforms import Form, StringField, SelectField, validators
+from wtforms.fields.html5 import EmailField
+from wtforms.validators import ValidationError
+
+VALID_COUNTRIES = ['usa', 'canada', 'portugal', 'uk', 'germany']
+FORM_KEYS = ['firstname', 'lastname', 'expert', 'email', 'phone', 'country']
+
+class ShifterInfoForm(Form):
+
+    def get_experts():
```

Comment:
This will only get run once at startup. I think the way you can do this is by initially populating the choices as an empty list like:

```python
expert = SelectField('Expert', choices=[])
```

and then populating it when you create the form:

```python
form = ShifterInfoForm(request.form)
form.expert.choices = get_experts()
```

I haven't tested it, but I think you should be able to do something similar.

Reply:
Hmm, not sure I understand this. I only want get_experts() run once at startup.

Reply:
I typically never have to restart the minard workers, so they can be running for months. In that time period if you remove or add an expert they won't show up. Admittedly that won't happen too often, but it'd be nice to not have to restart the flask app every time those change.

Reply:
Oh I see, that's not even called when the page is re-opened?

Reply:
It didn't in my testing. You can double check. I just added a print statement in there and navigated to the page a couple of times but only saw it print at startup.